### ISSUE TITLE
Fail to parse route when there are nhflags

### ISSUE THREAD

**Issue by pierresantana** [diosney/node-iproute#6]
As we can see in the data bellow, when we have nhflags on route, ip_route.show fail to parse the data.

```
$ ip route show table main
default via 192.168.0.1 dev eth0 onlink
169.254.0.0/16 dev eth0 scope link metric 1000
192.168.0.0/24 dev eth0 proto kernel scope link src 192.168.0.110
```

```
[ { type: 'default',
    to: 'via',
    '192.168.0.1': 'dev',
    eth0: 'onlink' },
  { type: 'unicast',
    to: '169.254.0.0/16',
    dev: 'eth0',
    scope: 'link',
    metric: '1000' },
  { type: 'unicast',
    to: '192.168.0.0/24',
    dev: 'eth0',
    proto: 'kernel',
    scope: 'link',
    src: '192.168.0.110' } ]
```

**Comment by diosney** [comment]
I just published version `2.0.0-beta`, which is an overhaul of the library and fixes a lot of issues, this one included, since it uses now the native `iproute -json` option to get parsed results out of the box in show methods, so you can consider it as fixed in `2.0`.

The beta tag is mainly to keep it a week or two for you to test it and let me know of anything before publishing the final version, but it should be fine.

One thing though, is that the field names in the resulting data come from `iproute` itself, and are different than in `1.x.x`, so keep that in mind if you end up using `2.0`.

Also, I need help completing the `show` interfaces, since there is no much documentation on it, so it could be nice a PR on that or at least an example output here so I can gather the missing interface keys.

**Comment by diosney** [comment]
You can test the beta by issuing `npm install iproute@beta`.
